app/assets/stylesheets/documentation_v2.scss

```diff
@@ -1,4 +1,4 @@
-// = require swagger-ui
+//= require swagger-ui
 
```

Comment:
Swagger UI is broken. This patch adds an empty file as an expedient since the gem hasn't been updated since 2015. We may need to fork it.

<img width="768" alt="Screen Shot 2019-11-22 at 5 47 31 PM" src="https://user-images.githubusercontent.com/4433943/69465870-66b67600-0d50-11ea-8506-8308c659809a.png">
<img width="703" alt="Screen Shot 2019-11-22 at 5 47 38 PM" src="https://user-images.githubusercontent.com/4433943/69465871-66b67600-0d50-11ea-8f3c-77fade641306.png">

Reply:
I am happy locking the documentation v2/v3 styles and js at this point - ie creating compiled versions of the css and js, and importing those, rather than gems, etc.

If we redesign the documentation, I'm happy figuring out a newer solution at that point, rather than extending what we have now.

(note: if we do make a compiled version, we should put a comment at the top with the URL of the PR where the compilation happens. This would have saved me headaches if I did it for `og_application.css`)

Reply:
Sweet — that'll save some time.

Not sure if by _compile_ you mean vendorize or vendorize + minify, but in case it's the latter — I had to un-minify `og_application.css` to track down some bugs in it — I think it's better if we let the asset pipeline handle the minification (or set it up to do so if it doesn't already).

Reply:
Agreed @jmromer, just vendorize - they shouldn't be minified, that was a bad choice on my part. I _would_ prefer it to all be in one file, or one directory, and have it clearly documented in the root js/css files what is happening, so that we know why it is the way it is, and don't accidentally use the vendorized code.